usr/src/cmd/text-install/osol_install/text_install/ti_install_utils.py

```diff
@@ -426,3 +443,62 @@ def get_zpool_list():
     zpool_list.extend(pool_list("status"))
     
     return zpool_list
+
+def get_zpool_free_size(name):
+    '''Return free size, available for pool's top-level filesystem.
+    Pool is imported if necessary
+    '''
+    
+    status = os.system("/usr/sbin/zpool list %s >/dev/null 2>&1" % (name))
```

Comment:
It's better to use subprocess.Popen instead of os.system. Even better would be to have a wrapper function. There was one exec_command, why not use that one?